assets/js/registration.js

```diff
@@ -0,0 +1,22 @@
+let timeZone = document.querySelectorAll(".num");
+
+let countDownDate = new Date("July 01, 2021 00:00:00").getTime();
```

Comment:
Why July 1 and not `new Date()`

Reply:
Here you can write any date for the countdown.